shopinvader/services/cart.py

```diff
@@ -45,9 +45,16 @@ def update(self, **params):
             return self._to_json(cart)
 
     def add_item(self, **params):
-        cart = self._get()
```

Comment:
NOTE: before this change the check `if not cart` was useless because the cart was always created if missing.